reporter/html.js

```diff
@@ -624,7 +624,13 @@ QUnit.testStart(function( details ) {
 
 	running = id( "qunit-testresult" );
 	if ( running ) {
-		running.innerHTML = "Running: <br />" + getNameHtml( details.name, details.module );
+		bad = QUnit.config.reorder && defined.sessionStorage &&
+			+sessionStorage.getItem( "qunit-test-" + details.module + "-" + details.name );
+
+		running.innerHTML = bad ?
+			"Rerunning previously failed test: <br />" :
```

Comment:
This needs parens. Currently the test name is only added to "Running", not to the new string.

Btw. I'm testing this by making an async test fail (change `expect()` call), then increasing the timeout to 2 seconds.